udata/forms/fields.py

```diff
@@ -200,10 +198,29 @@ def __getattr__(self, name):
 
 class FormField(FieldHelper, fields.FormField):
     def __init__(self, form_class, *args, **kwargs):
+        self.has_data = False
```

Comment:
The big problem is:
- WTForms is meant for form validation (so flat structure with MultiDict) and we are dealing with JSON and nested object
- WTForms consider FormField as required. It took me a lot of mailinglists/issues/stackoverflow reading to come to this "clean" solution :/

Conclusion: I think we should migrate from WTForms to something dedicated to JSON/dict